README.md

```diff
@@ -129,6 +129,16 @@ The workflow is the same as iterating on the Python files but needs additional s
 This will allow you to run `ninja install` after changes on the C++ files and update the "live" PyPRT package in the `pyprt` subdir.
 The `setup.py clean` call mentioned above will also clean out the native extension from the `pyprt` source directory.
 
+#### C++ Debug Builds
+
+By default, the native module is built in "release" mode (full optimization, no debug symbols). If the `--debug` argument is passed to `setup.py`, the CMake scripts will switch to "Release with Debug Info" mode, this will _disable optimization_ (`-O0`) to provide correct per-line debugging.
+
+To enable native debugging when building wheels or conda packages use:
+`python setup.py build --debug bdist_wheel` (or `bdist_conda`)
```

Comment:
As discussed, we need to add the "--skip-build" argument as well to avoid running the build command twice.

Reply:
done